### PR TITLE
block_writer: show lifetime queries/sec

### PR DESCRIPTION
**Pull request by petermattis**
Output lifetime queries/sec if the duration or max-blocks are limited.

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/examples-go/80)

### CODE REVIEW

File: block_writer/main.go [hunk 1]
```diff
@@ -243,6 +243,11 @@ func main() {
 			fmt.Printf("%6s: %6.1f/sec",
 				time.Duration(time.Since(start).Seconds()+0.5)*time.Second,
 				float64(dumps-lastNumDumps)/elapsed.Seconds())
+			if *duration > 0 || *maxBlocks > 0 {
+				// If the duration or max-blocks are limited, show the average
+				// performance since starting.
+				fmt.Printf("  %6.1f/sec", float64(dumps)/time.Since(start).Seconds())
```
Comment (tamird) [inline block_writer/main.go]: two spaces?

Comment (petermattis) [inline block_writer/main.go]: Intentional. It looked better.

**Comment by tamird** [comment]
'kay

**Comment by petermattis** [comment]
@jordanlewis What's up with the teamcity failure?

**Comment by jordanlewis** [comment]
I'm not sure what's going on with the `git pull` error there, but don't we need to update this repo after the move to `pkg`?

```
[11:59:21]package github.com/cockroachdb/cockroach/util/randutil: exit status 1
[11:59:21]package github.com/cockroachdb/cockroach/security: cannot find package "github.com/cockroachdb/cockroach/security" in any of:
[11:59:21]	/usr/local/go/src/github.com/cockroachdb/cockroach/security (from $GOROOT)
[11:59:21]	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/security (from $GOPATH)
[11:59:21]package github.com/cockroachdb/cockroach/security/securitytest: cannot find package "github.com/cockroachdb/cockroach/security/securitytest" in any of:
[11:59:21]	/usr/local/go/src/github.com/cockroachdb/cockroach/security/securitytest (from $GOROOT)
[11:59:21]	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/security/securitytest (from $GOPATH)
[11:59:22]package github.com/cockroachdb/cockroach/util/stop: cannot find package "github.com/cockroachdb/cockroach/util/stop" in any of:
[11:59:22]	/usr/local/go/src/github.com/cockroachdb/cockroach/util/stop (from $GOROOT)
[11:59:22]	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/util/stop (from $GOPATH)
```

**Pull request merged**